enigma-principal/app/src/epoch_u/epoch_provider.rs

```diff
@@ -98,14 +100,16 @@ impl EpochProvider {
         Ok(())
     }
 
-    #[logfn(DEBUG)]
+    //    #[logfn(DEBUG)]
```

Comment:
Is this comment left here on purpose?

Reply:
No, I commented it out to see a stack trace. Will fix.

Reply:
Fixed 4b73d61cb574b89eb255b1e9fde93d5fb34d383a